### DISCUSSION TITLE
getOrders - RestrictedResources error

### DISCUSSION THREAD

**Discussion by judev-jbg** [saleweaver/python-amazon-sp-api#710] [FAQ]
Hello:
I am using the **get_orders** endpoint and trying to access the restricted buyer information and shipping address RestrictedResources=['buyerInfo', 'shippingAddress'] however the response tells me that my application that does not have access to that information, but my application if configured to access.

`
Error code:
sp_api.base.exceptions.SellingApiBadRequestException: [{'code': 'InvalidInput', 'message': 'Application does not have access to one or more requested data elements: [buyerInfo, shippingAddress]', 'details': ''}]]
`

Image of my configuration in seller central (app)

![image](https://user-images.githubusercontent.com/45266431/186897641-45ad989a-e636-4d9e-aec6-a3c85b48e80d.png)

**Any idea what I am doing wrong?????**

**Comment by bowu8** [comment]
I have same error.  Any ideas?

**Comment by ghost** [comment]
I have same error. Any ideas?

**Comment by rugved-bodke-stigya** [comment]
I am unable to get the ARN field :( in the application

**Reply by ghost** [reply]
In the developer center, edit your developer profile and check all available options.
